lib/StatusIndicator/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const StatusIndicator = props => 
+  <span>
```

Comment:
We were speaking about styling inside components, do you think we should add some base styles? We kind of thought that styling should be something done in the app?